addons/core/translations/form/en-us.yaml

```diff
@@ -141,15 +141,15 @@ subscription_id:
   help: The subscription ID for the subscription with read access.
 access_key_id:
   label: Access Key ID
-  help: The access key ID found on the Security Credentials page in AWS.
+  help: The access key ID generated by AWS for the IAM user to use with this host catalog.
 secret_access_key:
   label: Secret Access Key
-  help: The secret access key found on the Security Credentials page in AWS.
+  help: The secret access key generated by AWS for the IAM user to use with this host catalog.
 region:
   label: Region
```

Comment:
Is this still being used or can it be removed?

Reply:
removed! thank you